### ISSUE TITLE
Should project.NewProject have default EnvironmentLookup?

### ISSUE THREAD

**Issue by weitenghuang** [docker/libcompose#372]
I am wondering why`project.NewProject()` function doesn't have default `EnvironmentLookup` while `docker.NewProject()` has [default value](https://github.com/docker/libcompose/blob/master/docker/project.go#L35).

This requires client project needs to copy code from libcompose if it uses `github.com/docker/libcompose/project`: https://github.com/skippbox/kompose/blob/master/pkg/loader/compose/compose.go#L105
```
	if context.EnvironmentLookup == nil {
		cwd, err := os.Getwd()
		if err != nil {
			return kobject.KomposeObject{}
		}
		context.EnvironmentLookup = &lookup.ComposableEnvLookup{
			Lookups: []config.EnvironmentLookup{
				&lookup.EnvfileLookup{
					Path: filepath.Join(cwd, ".env"),
				},
				&lookup.OsEnvLookup{},
			},
		}
	}
```

**Comment by vdemeester** [comment]
Hum right, there is a bunch of attributes we have default value in the `docker` package that we might be able to move in `project` as they (the defaults values) do not depend on docker.

/cc @joshwget

**Comment by joshwget** [comment]
Yeah, I definitely agree! `project` should have sensible defaults, especially since it's so easy to set these fields.

**Comment by weitenghuang** [comment]
resolved by #378
